cirq/ops/pauli_string_test.py

```diff
@@ -346,6 +436,31 @@ def test_pos():
     assert ps1 == +ps1
 
 
+def test_pow():
+    a, b = cirq.LineQubit.range(2)
+
+    assert cirq.PauliString({a: cirq.X})**0.25 == cirq.X(a)**0.25
+    assert cirq.PauliString({a: cirq.Y})**0.25 == cirq.Y(a)**0.25
+    assert cirq.PauliString({a: cirq.Z})**0.25 == cirq.Z(a)**0.25
+
+    p = cirq.PauliString({a: cirq.X, b: cirq.Y})
+    assert p**1 == p
+    assert p**-1 == p
```

Comment:
Unrelated: Could you please tell me why this is the case?  Why is the inverse of a PauliString acting like an identity operation?

Reply:
It's not acting like 1, it's acting like -1. (-1)**(-1) == -1

Reply:
If `==-1`, why is `p**-1 == p` ? I think i'm missing a `-1` somewhere, but i can't see it from the `p` that's been defined here.

Reply:
What's the inverse of the X gate? The X gate. X**-1 == X. This is true of Pauli products more generally.

Reply:
Ah! Thanks.

Reply:
IOW, (aX + bY + Z)^2 is identity (assuming [a, b, c] is a normalized vector in R^3). You can see this by writing out the 3-by-3 table of all products in this expression. Diagonal products are squares of Paulis, i.e. identity and off-diagonal elements can be paired up to cancel out since Paulis anti-commute.

Reply:
Thank you for the clarification.